src/common/commonutils/PackageUtils.c

```diff
@@ -3,12 +3,17 @@
 
 #include "Internal.h"
 
+#if ((defined(__GNUC__) && ((__GNUC__ > 4) || ((__GNUC__ == 4) && (__GNUC_MINOR__ >= 9)))) || defined(__clang__))
```

Comment:
For readability can we use named constants instead eg.

```c
#define USE_STD_ATOMIC ((defined(__GNUC__) && ((__GNUC__ > 4) || ((__GNUC__ == 4) && (__GNUC_MINOR__ >= 9)))) || defined(__clang__))

#if USE_STD_ATOMIC
#include <stdatomic.h>
#endif
```

Reply:
I like the proposal and we could move that USE_STD_ATOMIC macro in commonutils.h. This same pattern is in use in several places, is not new in here, and also is throw away code when we will go to gcc 10, and also other places may be deleted when we will cut deprecated parts. So will keep this feedback for applying it in a future PR. Thanks!